tests/Aspire.Playground.Tests/AppHostTests.cs

```diff
@@ -145,9 +145,22 @@ public async Task TestEndpointsReturnOk(TestEndpoints testEndpoints)
     public static TheoryData<string> AppHostAssemblies()
     {
         var appHostAssemblies = GetPlaygroundAppHostAssemblyPaths();
+
+        HashSet<string> appHostsWithTestEndpoints = new();
+        foreach (var testEndpoint in GetAllTestEndpoints())
+        {
+            appHostsWithTestEndpoints.Add(testEndpoint.AppHost);
+        }
+
         var theoryData = new TheoryData<string>();
         foreach (var asm in appHostAssemblies)
         {
+            var appHostName = Path.GetFileNameWithoutExtension(asm);
+            if (appHostsWithTestEndpoints.Contains(appHostName))
```

Comment:
Can you add a comment why we are filtering these?

Reply:
I added a comment, and renamed the method so it is clearer.